gerrit.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
we shouldn't be changing the files you didn't add.
2019 is appropriate, since these were added in 2019.